reference/speeduino.ini

```diff
@@ -3935,7 +3935,7 @@ cmdVSSratio6 =      "E\x99\x06"
    flex             = scalar,   U08,    34, "%",      1.000, 0.000
    flexFuelCor      = scalar,   U08,    35, "%",      1.000, 0.000
    flexIgnCor       = scalar,   S08,    36, "deg",    1.000, 0.000
-   idleLoad         = scalar,   U08,    37, { bitStringValue( idleUnits , iacAlgorithm  ) },    { (iacAlgorithm == 2 || iacAlgorithm == 3) ? 1.000 : 2.000 }, 0.000 ; This is a combined variable covering both PWM and stepper IACs. The units and precision used depend on which idle algorithm is chosen
+   idleLoad         = scalar,   U08,    37, { bitStringValue( idleUnits , iacAlgorithm  ) },    { (iacAlgorithm == 2 || iacAlgorithm == 3 || iacMaxSteps <= 85) ? 1.000 : 2.000 }, 0.000 ; This is a combined variable covering both PWM and stepper IACs. The units and precision used depend on which idle algorithm is chosen
```

Comment:
Should the iaxMaxSteps check here be `<=255`? I know the value is divided by 3 when sent to the arduino, but TS still views the value in its original form.